tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    int buffer_size = 16;
```

Comment:
Hm, VLAs should not be used if you can help it. I would rather use `enum { buffer_size = 16 };`.